### ISSUE TITLE
Port Forwarding is not working with Transmission

### ISSUE THREAD

**Issue by Baenwort** [glorious1/manual-connections#3]
So running Transmission 3.0 in FreeBSD 12.1 jail on latest repo .  I've got the script reporting success and setting the port forwarding port number in transmission (confirmed in GUI) however, the GUI still reports that the port is closed.

![image](https://user-images.githubusercontent.com/32050048/99916218-64be2480-2cce-11eb-8404-d5cb5924032c.png)

Here are the results of the port forwarding curl test you provide:

![image](https://user-images.githubusercontent.com/32050048/99916201-4fe19100-2cce-11eb-902a-3fc8e2804d10.png)

still working on figuring this out. 

My killswitch is functional using ipfw and I am transiting data through PIA but not showing as connectable.

**Comment by Feldegast123** [comment]
I rolled back to transmission 2.94 to get it to work again.

**Comment by Feldegast123** [comment]
Nevermind I was wrong. Got the same thing as you now.

**Comment by Feldegast123** [comment]
Ok now I think I solved it for me atleast. 

I'm guessing you have the same setup as me @Baenwort with that old IPFW-killswitch. 

I removed Openvpn from /etc/rc.conf and now it's working. I think I had two tunnels running and the port was getting opened in the wrong one.

**Comment by glorious1** [comment]
Again, sorry for the late reply, I never got notices of the issues.
I have found another problem that can lead to the scripts reporting success, but transmission reporting port is closed.  It can happen if you have added another tun.  Apparently it only works if openvpn is using tun0.  If that already exists, openvpn will create tun1 or higher and use that.

There is some logic in the original script I forked to kill the earlier openvpn process, which also kills the tun.  It only works interactively at present, I'm not sure what would happen if it were a cron job.
